package.json

```diff
@@ -8,7 +8,8 @@
     "test:watch": "jest --watchAll --notify",
     "build:epc": "bash generate.sh ",
     "build": "bash generate.sh && bash dist.sh",
-    "lint": "eslint epc-deck.js epc-graph.js epc-polyfill.js epc-table.js epc-test.js epc-ui.js"
+    "lint": "eslint epc-deck.js epc-graph.js epc-polyfill.js epc-table.js epc-test.js epc-ui.js",
```

Comment:
Could this use the `lint.sh` script, so we don't need to maintain the list of javascript files to be linted in multiple places?

Reply:
Good catch. I actually opted to nix several of the unnecessary lint scripts [here](https://github.com/matt-kimball/epc/pull/27/files), so this line is going away completely.